provisioner/template_test.go

```diff
@@ -464,3 +464,33 @@ DwIDAQAB
 	require.NoError(t, err)
 	require.EqualValues(t, pubkey, result)
 }
+
+func TestStupsNATSubnets(t *testing.T) {
+	for _, tc := range []struct {
+		vpc     string
+		subnets string
+	}{
+		{
+			vpc:     "172.31.0.0/16",
+			subnets: "172.31.64.0/28,172.31.64.16/28,172.31.64.32/28",
+		},
+		{
+			vpc:     "10.153.192.0/19",
+			subnets: "10.153.200.0/28,10.153.200.16/28,10.153.200.32/28",
+		},
+
+		{
+			vpc:     "10.149.64.0/19",
+			subnets: "10.149.72.0/28,10.149.72.16/28,10.149.72.32/28",
```

Comment:
Maybe error cases would be also good to check, such that we are sure that the user input won't trigger weird bugs.

Reply:
Forgot to add the tests. Done.